corehq/motech/fhir/models.py

```diff
@@ -31,11 +32,11 @@ class FHIRResourceType(models.Model):
     case_type = models.ForeignKey(CaseType, on_delete=models.CASCADE)
 
     # For a list of resource types, see http://hl7.org/fhir/resourcelist.html
-    name = models.CharField(max_length=255)
+    name = models.CharField(max_length=255, validators=[validate_supported_type])
 
     # `template` offers a way to define a FHIR resource if it cannot be
     # built using only mapped case properties.
-    template = JSONField(default=dict)
+    template = JSONField(default=dict, null=True, blank=True)
```

Comment:
Thank you. This has been annoying me.

I'm wondering ... do you think we should just drop this? We can achieve the same result using FHIRResourceProperties with constant values in the Admin UI. Maybe this is a case of YAGNI?

Reply:
you mean drop the column? I don't fully understand the use of this column, so I leave that call to you.

Reply:
Cool. I'll do that after this is merged, to avoid conflicts.

Reply:
@kaapstorm i just merged the base branch to master, so good for you to drop the column now, if still needed